spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
Yeah, to add onto what @RReverser is saying. You can't trust `directive` on an `ExpressionStatement` if there's even the possibility of transforming it. Whenever you wanted to replace an expression you'd have to check if the parent was an `ExpressionStatement` and invalidate it if it's a directive which is rather unnecessary.